app/presenters/concerns/document/field/entities.rb

```diff
@@ -4,50 +4,51 @@ module Document
   module Field
     ##
     # Methods for presenting entities (agents, concepts, etc)
+    # TODO: move to an EntitiePresenter class?
     module Entities
       ##
       # Retrieves the document's entities
       #
       # @param type [String,Symbol] name of entity group, e.g. "timespans"
-      # @param proxy_field [String,Symbol] name of field in proxies entities
-      #   are to be retrieved for, e.g. "dctermsTemporal"
+      # @param field [String,Symbol] name of field in the document entities
+      #   are to be retrieved for, e.g. "proxies.dctermsTemporal"
       # @return [Array] document's entities
-      def entities(type, proxy_field = nil)
+      def document_entities(type, field = nil)
         @entities ||= {}
         @entities[type] ||= {}
-        @entities[type][proxy_field] ||= entities_for(type, proxy_field)
+        @entities[type][field] ||= document_entities_for_type(type, field)
       end
 
-      # @param (see #entities)
-      def entities_for(type, proxy_field = nil)
+      # @param (see #document_entities)
+      def document_entities_for_type(type, field = nil)
         document_entities = document.fetch(type, [])
```

Comment:
Is that a problem? As there is no method `#document_entities=`, this will always result in a local variable instantiation.

Reply:
not a problem from a functional point of view, just when I was reading it initially it was confusing since:
`document_entities` calls `document_entities_for_type` which at the end returns `document_entities`

Reply:
Renamed in 92e2dce0